lib/kino/vega_lite.ex

```diff
@@ -86,6 +86,32 @@ defmodule Kino.VegaLite do
     Kino.JS.Live.cast(kino, {:push, dataset, data_points, window})
   end
 
+  @doc """
+  Updates a vega-lite [parameter's](https://vega.github.io/vega-lite/docs/parameter.html#variable-parameters) value.
+
+  The parameter must be registered: `VegaLite.param(vl, "param_name", opts)`.
+
+  To use the parameter in the chart, set a property to `[expr: "param_name"]`.
+
+  ## Examples
+
+      chart =
+        VegaLite.new(width: 400, height: 400)
+        |> VegaLite.param("strokeWidth", value: 3)
+        |> VegaLite.mark(:line, stroke_width: [expr: "strokeWidth"])
+        |> VegaLite.encode_field(:x, "x", type: :quantitative)
+        |> VegaLite.encode_field(:y, "y", type: :quantitative)
+        |> Kino.VegaLite.new()
+        |> Kino.render()
+
+      Kino.VegaLite.set_param(chart, "strokeWidth", 10)
```

Comment:
Should work just as good, and we stay consistent?

```suggestion
      chart =
        VegaLite.new(width: 400, height: 400)
        |> VegaLite.param("stroke_width", value: 3)
        |> VegaLite.mark(:line, stroke_width: [expr: "stroke_width"])
        |> VegaLite.encode_field(:x, "x", type: :quantitative)
        |> VegaLite.encode_field(:y, "y", type: :quantitative)
        |> Kino.VegaLite.new()
        |> Kino.render()

      Kino.VegaLite.set_param(chart, "stroke_width", 10)
```